WordPress/gutenberg
# Add command to navigate to site editor

## What?

Enhancement for the scenario mentioned in https://github.com/WordPress/gutenberg/issues/61460

## Why?

Currently there is no command option to go back to site editor from edit post or page views.


## How?

Added command to navigate to Site Editor.

## Testing Instructions

- Open the WordPress Editor
- Go to Posts or Pages in the WordPress admin.
- Open a new or existing post/page to enter the block editor.
- Open the Command Palette
- Access the Command Palette by pressing Cmd + K on Mac or Ctrl + K on Windows, or via the menu if available.
- In the Command Palette, look for a command labeled something like "Go to Site Editor".
- Ensure that this command is listed among the available commands.
- Execute the Command
- Select the Go to Site Editor command from the Command Palette.
- Confirm that executing the command redirects you to the Site Editor interface.



## Screenshots or screencast 

https://github.com/user-attachments/assets/e39219d2-195f-4eda-80ce-2ddc73eb6c14
